### ISSUE TITLE
Are apps made for genuine Philips Hue meant to be able to detect HA-bridge devices?

### ISSUE THREAD

**Issue by Jinsterrrr** [bwssytems/ha-bridge#601]
Hey guys. My first post on github. Basically, I have setup Echo with the help of HA-bridge to control my limitless/milight LED downlights. Echo can detect HA-bridge masked devices very well - the lights come up with Hue LUX.

But none of the apps/programs made for Philips Hue can pick up HA-bridge as a Hue bridge.

My HA-bridge is on Port 80.

Is this normal behaviour? Should the genuine Philips or other apps made for genuine Hue be able to detect HA-bridge as a Hue Bridge?

**Comment by audiofreak9** [comment]
The "HUE" Apps do detect the ha-bridge, and for most of the apps do require port 80.

Read this FAQ: https://github.com/bwssytems/ha-bridge/wiki/HA-Bridge-FAQs#apps
And this FAQ: https://github.com/bwssytems/ha-bridge/wiki/HA-Bridge-FAQs#HUEReleaseVersion

**Comment by tuicemen** [comment]
I had never thought of using one of these apps till now.
However I also can't get anyhue apps to work with the bridge.
I suspect it is because I now run the Bridge in secure mode & or I don't have a hue hub & or my phone is to old.

**Comment by bwssytems** [comment]
The big issue for the apps is that they have to have the manual setup feature. Any Hue App tries to "phone home" to meethue.com first. Since this not a real philips device, that does not work. The last version of the hue app had a manual setup.

**Comment by tuicemen** [comment]
Well that Explains my problem.
the Phone is too old to run the new Hue app.

**Comment by Jinsterrrr** [comment]
Aha. That worked. The latest official Philips Hue app on iOS picked up my HA-bridge once I pressed the Help button, which allows you to manually search. Interestingly, it picked up the HA-bridge before I actually entered an IP address.

There is a new issue now, that the slides in the Hue app, instead of changing the brightness of the lights, changes the temperature for Group 1 and Group 2 of Milights simultaneously, and in opposite ways....

So, when I slide the intensity up on Group 1 Milights, instead of changing brightness of Group 1, what actually happens is: Group 1 gets cooler white and Group 2 gets warmer white. And when I slide the Group 1 intensity down, Group 1 gets warmer white and Group 2 get cooler white.

The on/off switches do work properly.
